Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -633,7 +782,8 @@ void getElementIntegerVariables(
 
     for (long i = 0; i < nn; i++)
     {
-        elem_info[counter++] = local_node_ids.at(elem.getNodeIndex(i));
+        auto const& n = *elem.getNode(i);
+        elem_info[counter++] = local_node_ids.at(n.getID());
```

Comment:
I wonder why is the access into `local_node_ids` via the `at()` method?

Reply:
For safety. In a map, accessed via `op[]`, a default value would be created.

Reply:
Note to myself: local_node_ids could be passed as `const&` maybe...

Reply:
Thanks for the explanation. I overlook that `local_node_ids` is a `map`.